src/libraries/shims/src/mscorlib.forwards.cs

```diff
@@ -19,6 +19,95 @@
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.OrdinalComparer))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.UnitySerializationHolder))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Diagnostics.Contracts.ContractException))]
-// This is required for back-compatibility with legacy Xamarin which had Stack<T> and Queue<T> in mscorlib
-[assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.Stack<>))]
+// This is required for back-compatibility with legacy Xamarin which had Stack<T> and Queue<T> in mscorlib and also added a few NS2.1 types that didn't exist in .NET Framework
```

Comment:
Did you want to add an APICompat run with Xamarin somewhere to capture the differences?  We do similar things in https://github.com/dotnet/runtime/blob/d3b7dc1399d0214d627a4197be1fab734a84e05f/src/libraries/apicompat/ApiCompat.proj#L43.

Reply:
I looked at that but we'd need the Xamarin assemblies somewhere (e.g. a nuget package) and we don't have that as far as I know, I don't think we need it given these assemblies will never change.